.changeset/config.yaml

```diff
@@ -58,14 +58,14 @@ types:
     hide: true
 
 modules:
-  - id: x/fees
-    description: Fees
   - id: x/profiles
     description: Profiles
   - id: x/subspaces
     description: Subspaces
   - id: x/relationships
     description: Relationships
+  - id: x/fees
+    description: Fees
```

Comment:
This change is just to make sure that the `x/fees` module is listed below more important modules inside the changelog since `changeset` bases the entries on the order in which modules are listed